SMLHelper/Handlers/LootDistributionHandler.cs

```diff
@@ -0,0 +1,145 @@
+﻿namespace SMLHelper.V2.Handlers
+{
+    using Interfaces;
+    using Patchers;
+    using SMLHelper.V2.Assets;
+    using System.Collections.Generic;
+    using System.Linq;
+    using UWE;
+
+    /// <summary>
+    /// A handler that manages Loot Distribution.
+    /// </summary>
+    public class LootDistributionHandler : ILootDistributionHandler
+    {
+        /// <summary>
+        /// Main entry point for all calls to this handler.
+        /// </summary>
+        public static ILootDistributionHandler Main { get; } = new LootDistributionHandler();
+
+        private LootDistributionHandler() { } // Hides constructor
+
+        /// <summary>
+        /// Adds in a custom entry into the Loot Distribution of the game.
+        /// You must also add the <see cref="WorldEntityInfo"/> into the <see cref="WorldEntityDatabase"/> using <see cref="WorldEntityDatabaseHandler"/>.
+        /// </summary>
+        /// <param name="data">The <see cref="LootDistributionData.SrcData"/> that contains data related to the spawning of a prefab, also contains the path to the prefab.</param>
+        /// <param name="classId">The classId of the prefab.</param>
+        /// <param name="info">The WorldEntityInfo of the prefab. For more information on how to set this up, see <see cref="WorldEntityDatabaseHandler"/>.</param>
+        public static void AddLootDistributionData(string classId, LootDistributionData.SrcData data, WorldEntityInfo info = null)
+        {
+            Main.AddLootDistributionData(data, classId);
+
+            if (info != null)
+                WorldEntityDatabaseHandler.AddCustomInfo(classId, info);
```

Comment:
Why is this present in `LootDistributionHandler.AddLootDistributionData` but not in `LootDistributionHandler.Main.AddLootDistributionData`? Shouldn't static methods behave _exactly_ the same as instance methods in the case of dependency injection? (So call them without doing any additional stuff)